fuel-tx/src/transaction/fee.rs

```diff
@@ -308,23 +426,4 @@ mod tests {
 
         assert!(overflow);
     }
-
-    #[test]
-    fn base_fee_gas_limit_less_than_gas_used_by_predicates() {
-        let metered_bytes = 5;
-        let gas_used_by_predicates = 8;
-        let gas_limit = 7;
-        let gas_price = 11;
-
-        let fee = TransactionFee::checked_from_values(
-            &PARAMS,
-            metered_bytes,
-            gas_used_by_predicates,
-            gas_limit,
-            gas_price,
-        )
-        .expect("failed to calculate fee");
-
-        assert!(fee.min_fee > fee.max_fee);
-    }
```

Comment:
I am removing this test because it is now no longer possible for `min_fee` to be greater than `max_fee`.